test/benchmarks/material/checkbox/app.module.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {A11yModule} from '@angular/cdk/a11y';
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {MatCheckboxModule} from '@angular/material/checkbox';
+
+/**
+ * @title Checkbox benchmark component.
+ */
+@Component({
+  selector: 'app-root',
+  template: `
+    <button id="show" (click)="show()">Show</button>
+    <button id="hide" (click)="hide()">Hide</button>
+    <button id="indeterminate" (click)="indeterminate()">Indeterminate</button>
+
+    <ng-container *ngIf="isVisible">
+      <mat-checkbox
+        [checked]="isChecked"
+        [indeterminate]="isIndeterminate"
+        (change)="toggleIsChecked()"
+        >Check me!</mat-checkbox>
+    </ng-container>
+  `,
+  encapsulation: ViewEncapsulation.None,
+  styleUrls: ['//src/material/core/theming/prebuilt/indigo-pink.css'],
```

Comment:
Did you confirm if that actually works? I wouldn't know of any resolution code that supports Bazel labels.

Reply:
@devversion Yeah, it works. Removing it breaks the styles. Kinda strange now that you mention it... @jelbourn Does `styleUrls` supports Bazel labels? lol

Reply:
I have no idea how this would work. @devversion and @wagnermaciel could you pair to figure out the best way to set this up?

Reply:
Did you think about the impacts of using `styleUrls` for that? couldn't it affect benchmarks in creation phase as the theme is actually loading deferred as part of the runtime JS?

Reply:
Just checked. It works _only_ with `ngtsc` (so not with View Engine) because ngtsc normalizes the path so that double-slash (as per Bazel label) becomes a path. That path is then matched against `rootDirs` of the TS program. `bazel-out` is a root directory, so the prebuilt theme can be resolved. That is reasonable (with the downside of not working for --config=view-engine).

Regardless of that though, I think we should not load the theme as part of the component factory. That will cause deviations in benchmarks as the theme will be loaded deferred and through Angular. I still think we should implement asset injection.

I think we could merge this in the interim. Asset injection would require upstream changes in `dev-infra`. @wagnermaciel Happy to jump on a call to explain what I refer to with asset injection

Reply:
That explains it! Yeah, I agree that we can leave this for now and change it to an asset in a follow-up. @wagnermaciel can you add a TODO and file an issue for yourself?

Reply:
Done and done https://github.com/angular/angular/issues/37445